bigquery/samples/tests/test_delete_table.py

```diff
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .. import create_table
+from .. import delete_table
 
 
-def test_create_table(capsys, client, random_table_id):
```

Comment:
Looks like we need to copy the `test_create_table` test to a `test_create_table.py` file so that we don't lose it.

Reply:
Opps, accidentally deleted this file. Adding it back now!